build/commands/scripts/format.js

```diff
@@ -238,4 +263,5 @@ const runMojomFormat = async (files, dryRun) => {
   return mojomFormatIssues
 }
 
+program.parse(process.argv)
```

Comment:
if fact, nobody has noticed this. The bad thing is we skip passing `--presubmit` when call `npm run format`